docs/UserGuide.md

```diff
@@ -1,186 +1,205 @@
+3. [Features](#features)
+4. [FAQ](#faq)
+5. [Storage file format](#storage-file-format)
 
-1. [Quick Start](#quick-start)
-2. [Features](#features)
-3. [FAQ](#faq)
-4. [Command Summary](#command-summary)
+## 1. Introduction 
+Welcome. We will take you for an enthralling journey as we bring forward our task manager `TaskCrusher`.
 
-## 1. Quick Start
+It can help you manage events, deadlines, add tasks with or without deadlines or view your history. This is a great manager if you are looking to move away from clicking as the simplified scheduling is just a few short and sweet command lines away.
+
+## 2. Quick Start
 
 0. Ensure you have Java version `1.8.0_60` or later installed in your Computer.<br>
 
    > Having any Java 8 version is not enough. <br>
    > This app will not work with earlier versions of Java 8.
 
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
+1. Download the latest `TaskCrusher.jar` from the [releases](../../../releases) tab.
+2. Copy the file to the folder you want to use as the home folder for TaskCrusher.
 3. Double-click the file to start the app. The GUI should appear in a few seconds.
-   > <img src="images/Ui.png" width="600">
-
 4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
    e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window.
+   
 5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` :
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
-6. Refer to the [Features](#features) section below for details of each command.<br>
+   * **`help`** : displays help documentation
+   * **`adde`:**  `add "Cs2103 meeting" 3 Mar 17:00 to 18:00` adds an event called "Cs2103 meeting" on 3 March from 17:00 to 18:00
+   * **`view`**: `view calendar` displays the calendar with tasks and events you have added
 
+6. Refer to the [Features](#features) section below for details of each command.<br>
 
-## 2. Features
+## 3. Features
 
 > **Command Format**
 >
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
+> * Items surrounding `|` means we only choose one of them (OR).
 > * Items with `...` after them can have multiple instances.
-> * Parameters can be in any order.
+> * Parameters should be in the order stated (at least for now).
 
-### 2.1. Viewing help : `help`
+### 3.1. Viewing help : `help`
 
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 3.2. Adding an item: `adde` or `addt`
+
+Format (event):
+
+`adde "event name" START_DATE START_TIME to [END_DATE] END_TIME [t/TAG]... [//DESCRIPTION]`<br>
```

Comment:
think how user friendly is `adde | addt` vs `add`? 
try to make your commands as user friendly as possible.